### PR TITLE
OEL-309: Add OE Whitelabel theme scaffold files.

### PR DESCRIPTION
**Pull request by 10atm-blue**
PR goals:
- Add Whitelabel theme file structure

### CODE REVIEW

**Review by drishu — COMMENTED** [review]
https://github.com/openeuropa/oe_whitelabel/issues/2 <-- click me

**Review by claudiu-cristea — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Review by drishu — CHANGES_REQUESTED** [review]
was to fast

**Review by claudiu-cristea — CHANGES_REQUESTED** [review]
Undo approval.

**Review by drishu — DISMISSED** [review]

File: .phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":[],"times":[]}
```
Comment (abel-santos-corral) [inline .phpunit.result.cache]: This file must be eliminated.

**Review by abel-santos-corral — APPROVED** [review]
Reviewed all files. File .phpunit.result.cache eliminated from git and added to .gitignore file. Also checked the assets.

**Review by drishu — APPROVED** [review]

**Review by claudiu-cristea — APPROVED** [review]

**Pull request merged**